### PR TITLE
add nexpy/nexusformat python package to utilities page 

### LINKED ISSUE #810: add nexpy/nexusformat python package to utilities page
https://github.com/nexpy/nexusformat

### PR DESCRIPTION
**Pull request by prjemian**
fixes #810

### CODE REVIEW

File: manual/source/utilities.rst [hunk 1]
```diff
@@ -141,41 +141,36 @@ The list is not intended to be a complete list of all available packages.
...
+
+   See the program documentation for more details:
    https://github.com/nexusformat/cnxvalidate.git
 
 .. index::
         validation
    file; validate
    punx (utility)
 
-.. _punx:
```
Comment (PeterC-DLS) [inline manual/source/utilities.rst:164]: Is hyperlink target unused?

File: manual/source/utilities.rst [hunk 2]
```diff
@@ -438,14 +445,29 @@ Language API: *Python*
...
+
 * **Mantis** [#]_, with NeXus-specific python code [#]_, reads :ref:`NXstxm`
 
   .. [#] Mantis: http://spectromicroscopy.com/
   .. [#] python code: https://bitbucket.org/mlerotic/spectromicroscopy/src/default/
 
+.. index:: API; Python; nexusformat
+
+* **nexusformat** [#]_ NeXus package for Python
+    Provides an API to open, create, and manipulate NeXus data.
```
Comment (PeterC-DLS) [inline manual/source/utilities.rst:465]: and plot

**Review by PeterC-DLS — CHANGES_REQUESTED** [review]

File: manual/source/utilities.rst (hunk 1 above)
Comment (prjemian) [reply manual/source/utilities.rst:164]: Adding those links where it was used in the h5py examples.  Moving the hyperlink target from validation.rst to here.

File: manual/source/utilities.rst (hunk 2 above)
Comment (prjemian) [reply manual/source/utilities.rst:465]: That's a good addition for the nexpy/nexusformat project.

File: manual/source/utilities.rst (hunk 1 above)
Comment (prjemian) [reply manual/source/utilities.rst:164]: 2nd thought, leave the hyperlink target in validations

**Review by PeterC-DLS — APPROVED** [review]

**Review by rayosborn — APPROVED** [review]

**Pull request merged**